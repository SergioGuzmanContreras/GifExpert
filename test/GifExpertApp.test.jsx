import { render, screen } from "@testing-library/react";
import { GifExpertApp } from "../src/GifExpertApp";

describe('Pruebas de <GifExpertApp />', () => { 

    test('Debe hacer match con el snapshot', ()=>{
        const {container} = render(<GifExpertApp />);
        expect(container).toMatchSnapshot();
    });

});