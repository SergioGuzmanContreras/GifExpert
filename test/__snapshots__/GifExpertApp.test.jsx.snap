// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas de <GifExpertApp />  1`] = `
<div>
  <h1>
    GifExpert
  </h1>
  <form
    aria-label="form"
  >
    <input
      placeholder="Buscar Gifs"
      type="text"
      value=""
    />
  </form>
  <h3>
    One Punch Man
  </h3>
  <h2>
    Cargando...
  </h2>
  <div
    class="card-grid"
  />
</div>
`;

exports[`Pruebas de <GifExpertApp /> Debe hacer match con el snapshot 1`] = `
<div>
  <h1>
    GifExpert
  </h1>
  <form
    aria-label="form"
  >
    <input
      placeholder="Buscar Gifs"
      type="text"
      value=""
    />
  </form>
  <h3>
    One Punch Man
  </h3>
  <h2>
    Cargando...
  </h2>
  <div
    class="card-grid"
  />
</div>
`;
